pkg/cli/admin/release/extract_tools.go

```diff
@@ -209,6 +209,27 @@ var (
 	program is [located on github](https://github.com/openshift/cloud-credential-operator).
 	`)
 
+	readmeOCMirror = heredoc.Doc(`
+	# OC Mirror Utility
+
+	The oc-mirror tool assists with mirroring OpenShift release content from a target
+	registry to another for disconnected install scenarios.
+
+	To learn more about OpenShift, visit [docs.openshift.com](https://docs.openshift.com)
+	and select the version of OpenShift you are using.
+
+	# Installing the tools
```

Comment:
IMO this should be `Installing the Tools`

Reply:
I copied from the other instances which are all "Installing the tools" but I can change it, no problem.